src/components/config-list/index.js

```diff
@@ -1,48 +1,32 @@
 import { Table, Typography } from '@equinor/eds-core-react';
 import React from 'react';
 
-import configHandler from '../../utils/config';
-
-const configSorter = (a, b) => a.key.localeCompare(b.key);
+import { configVariables } from '../../utils/config';
+import { keys } from '../../utils/config/keys';
 
 export const ConfigList = () => {
-  const configKeys = configHandler
-    .getConfigValues()
-    .sort(configSorter)
+  const configValues = Object.keys(configVariables)
+    .sort((a, b) => a.localeCompare(b))
     .map((c) => {
+      const value = configVariables[c];
       return (
-        <Table.Row key={c.key}>
+        <Table.Row key={c}>
           <Table.Cell>
-            <Typography variant="body_short">{c.key}</Typography>
+            <Typography>{c}</Typography>
           </Table.Cell>
           <Table.Cell>
             <pre>
               <Typography variant="body_short">
-                {JSON.stringify(c.value, null, 2)}
+                {c === keys.CLUSTER_EGRESS_IPS
```

Comment:
Would probably be better to transform the values when configVariables is built. Could create a "transformers" map in config/index.js where key is one of the keys defines in keys.js and a transformer function that takes a string and returns the transformed value, e.g. an array. The reduce function could then check if a transformer function is defined for the specific key and call this. If no transformer is defined, just use the original value.
In the config component you can then display the values based on the value type. For example, a value of type array could be displayed the way you handle EGRESS_IPS, and we don't need to hardcode how different variables should be displayed.

config/index.js could lokk something like this:

``` javascript
import * as configKeys from './keys';
import * as jsonConfig from '../../config.json';

const arrayTransformer = (value, delimiter = ',') =>
  value ? value.split(delimiter) : [];

const transformers = {
  [configKeys.keys.CLUSTER_EGRESS_IPS]: arrayTransformer,
};

export const configVariables = Object.keys(configKeys.keys)
  .map((key) => configKeys.keys[key])
  .reduce((config, key) => {
    const tranformer = transformers[key];
    const value =
      !window[key] || window[key].startsWith('${')
        ? jsonConfig.default[key]
        : window[key];
    config[key] = tranformer ? tranformer(value) : value;
    return config;
  }, {});
```

and config-list/index.js would then be

``` html
              <Typography variant="body_short">
                {Array.isArray(value)
                  ? value.join('\n')
                  : JSON.stringify(value, null, 2)}
              </Typography>
```

Reply:
good idea, added this.